Libraries/Components/Pressable/Pressable.js

```diff
@@ -210,17 +240,44 @@ function Pressable(props: Props, forwardedRef): React.Node {
   );
   const eventHandlers = usePressability(config);
 
+  const pressableTVEventHandler = (evt: Event) => {
+    if (props.isTVSelectable !== false || props.focusable !== false) {
+      const tag = findNodeHandle(viewRef.current);
+      if (tag === evt.tag) {
```

Comment:
Ah, knowing that you are just checking for whether it matches, if you find you need this, we added `event.target` that is a ref to the HostComponent.

You should be able to do: `if (viewRef.current === evt.target)`

Check out ScrollResponder for an example: https://github.com/facebook/react-native/blob/master/Libraries/Components/ScrollResponder.js#L190

Reply:
This change went out as part of 0.63: https://github.com/facebook/react-native/commit/6286270e4cb10b40cfd7c8193e31d965f6815150

Reply:
That's fantastic.... I'll do an audit of some other components too to see where we can convert to using evt.target